src/wrapper/Wrapper.jsx

```diff
@@ -35,7 +36,7 @@ const Wrapper = () => {
           setType,
           connect_to_user_wallet,
           metamaskEvent,
-          setMetamaskEvent
+          setMetamaskEvent,stopLoos_Contract,setstopLoos_Contract
```

Comment:
Let's correct the typo here and everywhere else ?
stopLoos_Contract -> **stoploss_contract**